### DISCUSSION TITLE
How to enable Comments for a new custom module

### DISCUSSION THREAD

**Discussion by copyyour** [YetiForceCompany/YetiForceCRM#15108] [How to]
Description

I have created a new module using "Create Module". Now I would like to allow Comments for that module. How can I enable comments?

**Comment by rskrzypczak** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/issues/10626
https://github.com/YetiForceCompany/YetiForceCRM/issues/10626#issuecomment-820134146

**Reply by copyyour** [reply]
I replaced this file LettersIn.php, but there is still no comment option in the module layout, please advise, thank you

**Reply by rskrzypczak** [reply]
I don't think we fully understand each other.
There is no need to change any files.
It shows the code that should be run for your module.

```php
$modcommentsModuleInstance = vtlib\Module::getInstance('ModComments'); 
 if ($modcommentsModuleInstance && file_exists('modules/ModComments/ModComments.php')) { 
 	include_once 'modules/ModComments/ModComments.php'; 
 	if (class_exists('ModComments')) { 
 		ModComments::addWidgetTo(['**YourCustomModuleName**']); 
 	} 
 } 
```

**Reply by copyyour** [reply]
Thank you very much, but what should I do if I have multiple custom modules that need to add comments, thank you

**Comment by copyyour** [comment]
I have changed the name of my module, but there is still no comment option in the module layout. Do I need other operations? I am a beginner. Thank you very much for your guidance.

			$modcommentsModuleInstance = vtlib\Module::getInstance('ModComments');
			if ($modcommentsModuleInstance && file_exists('modules/ModComments/ModComments.php')) {
				include_once 'modules/ModComments/ModComments.php';
				if (class_exists('ModComments')) {
					ModComments::addWidgetTo(['Drgsmbcs']);
				}
